convert2rhel/toolopts.py

```diff
@@ -15,6 +15,7 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+from builtins import object
```

Comment:
When building a pkg on both RHEL 6 and 7.
```
Traceback (most recent call last):
  File "setup.py", line 22, in <module>
    from man.build_manpage import build_manpage
  File "/builddir/build/BUILD/convert2rhel-v0.11/man/__init__.py", line 18, in <module>
    from convert2rhel import toolopts
  File "/builddir/build/BUILD/convert2rhel-v0.11/convert2rhel/toolopts.py", line 18, in <module>
    from builtins import object
ImportError: No module named builtins
```
https://download.copr.fedorainfracloud.org/results/@oamg/convert2rhel/epel-6-x86_64/01554887-convert2rhel/build.log.gz

Reply:
I've removed all the `builtins` imports. @SpyTec, were you adding them for any specific reason?
The thing is that the module is not available on CentOS/RHEL 6 and 7.

Reply:
In our case I don't think we need it, there is a reason for it being there though https://stackoverflow.com/a/36517807

Wouldn't builtins be available through pip?

Reply:
An rpm package can only depend on other rpms. If it's not packaged in CentOS/OL, we can't import/depend on it.

Reply:
Ah okay. I think we should be fine. If we need anything we can get around it by creating our own methods for it - luckily not too difficult